test/integration_tests/source_separation_pipeline_test.py

```diff
@@ -1,37 +1,41 @@
 import os
 import sys
 
+import pytest
 import torch
 import torchaudio
-from torchaudio.prototype.pipelines import CONVTASNET_BASE_LIBRI2MIX
+from torchaudio.prototype.pipelines import CONVTASNET_BASE_LIBRI2MIX, HDEMUCS_HIGH_MUSDB_PLUS
 
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "examples"))
-from source_separation.utils.metrics import PIT, sdr
+from source_separation.utils.metrics import sdr
 
 
-def test_source_separation_models(mixture_source, clean_sources):
+@pytest.mark.parametrize(
+    "bundle,task,channel,expected_score",
+    [
+        [CONVTASNET_BASE_LIBRI2MIX, "speech_separation", 1, 8.1373],
+        [HDEMUCS_HIGH_MUSDB_PLUS, "music_separation", 2, 8.7480],
+    ],
+)
+def test_source_separation_models(bundle, task, channel, expected_score, mixture_source, clean_sources):
     """Integration test for the source separation pipeline.
-    Given the mixture waveform with dimensions `(batch, 1, time)`, the pre-trained pipeline generates
+    Given the mixture waveform with dimensions `(batch, channel, time)`, the pre-trained pipeline generates
     the separated sources Tensor with dimensions `(batch, num_sources, time)`.
-    The test computes the scale-invariant signal-to-distortion ratio (Si-SDR) score in decibel (dB) with
-    permutation invariant training (PIT) criterion. PIT computes Si-SDR scores between the estimated sources and the
-    target sources for all permuations, then returns the highest values as the final output. The final
-    Si-SDR score should be equal to or larger than the expected score.
+    The test computes the signal-to-distortion ratio (SDR) score in decibel (dB).
```

Comment:
The score is actually Si-SDR although the function name is `sdr`. maybe we should change the function name later